packages/collector/test/tracing/frameworks/express/app.mjs

```diff
@@ -13,6 +13,13 @@ process.on('SIGTERM', () => {
   process.exit(0);
 });
 
+import mock from 'mock-require';
```

Comment:
issue: As far as I know `mock-require` does not work for esm apps. See https://jsw.ibm.com/browse/INSTA-788.

> Require-mock NOT working with ESM apps

Reply:
Thanks for pointing this, I haven't noticed the issue before 👍

Reply:
Updated the code.